connectors/es/sink.py

```diff
@@ -461,18 +459,18 @@ class ElasticServer(ESClient):
 
     - grabs all ids on Elasticsearch for the index
     - creates a MemQueue to hold documents to stream
-    - runs a `Fetcher` (producer) and a `Bulker` (consumer) against the queue
+    - runs a `Extractor` (producer) and a `Sink` (consumer) against the queue
```

Comment:
We could also name both `DocsProducer` and `DocsConsumer`, but `Sink` is definitely better than `Bulker`, so also happy with that option.